snapyr-samples/sample/src/main/java/com/snapyr/sdk/sample/SampleApp.java

```diff
@@ -63,9 +63,12 @@ public void onVmViolation(Violation v) {
                                         // to
                                         // client code in the wild. If you want to be really careful
                                         // uncomment the penaltyDeath line above --BS
-                                        Log.e(
-                                                v.getLocalizedMessage(),
-                                                String.valueOf(v.getCause().getStackTrace()));
+                                        Throwable cause = v.getCause();
+                                        String stack = "<cause was null>";
```

Comment:
did you mean to check this in?

Reply:
Yes - `v.getCause()` is null sometimes which was crashing the sample app with a null pointer exception. (just the sample app but yeah) Good looking out!